src/main/java/gregtech/loaders/recipe/handlers/PartsRecipeHandler.java

```diff
@@ -254,11 +254,19 @@ public static void processPlateDouble(OrePrefix doublePrefix, Material material,
                         "h", "P", "P", 'P', new UnificationEntry(plate, material));
             }
 
-            BENDER_RECIPES.recipeBuilder().EUt(VA[LV]).duration((int) material.getMass() * 2)
+            BENDER_RECIPES.recipeBuilder().EUt(96).duration((int) material.getMass() * 2)
                     .input(plate, material, 2)
                     .output(doublePrefix, material)
                     .circuitMeta(2)
                     .buildAndRegister();
```

Comment:
For OCD reasons these method calls could be the same order as everything else in the file.